src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserRegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static User user;
+    private static String DEFAULT_LOGIN = "login";
+    private static String DEFAULT_PASSWORD = "password";
+    private static Integer DEFAULT_AGE = 18;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void init() {
+        user = new User();
+        user.setLogin(DEFAULT_LOGIN);
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
+    }
+
+    @Test
+    void register_savingWithSameLogin_NotOk() {
+        user.setLogin("nice_login");
```

Comment:
The test `register_savingWithSameLogin_NotOk` sets the login to 'nice_login' but does not reset it to the default login before the second registration attempt. This might not accurately test the scenario of registering a user with the same login twice. Consider resetting the login to the same value before the second registration attempt.